platform/macos/export/export_plugin.cpp

```diff
@@ -2051,10 +2051,12 @@ bool EditorExportPlatformMacOS::has_valid_export_configuration(const Ref<EditorE
 	String architecture = p_preset->get("binary_format/architecture");
 	if (architecture == "universal" || architecture == "x86_64") {
 		if (!ResourceImporterTextureSettings::should_import_s3tc_bptc()) {
+			err += TTR("Cannot export for universal or x86_64 if S3TC BPTC texture format is disabled. Enable it in project settings.") + "\n";
```

Comment:
```suggestion
			err += TTR("Cannot export for universal or x86_64 if S3TC BPTC texture format is disabled. Enable it in the Project Settings.") + "\n";
```
Following the same style as other messages, should probably also instruct how to find them